### PR TITLE
[python/ru-ru] Corrections for Russian localization

### PR DESCRIPTION
**Pull request by ademaro**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by ademaro** [comment]
@Menelion Is there anything else you need to get PR accepted?

**Comment by ademaro** [comment]
@mribeirodantas, maybe you can help accept this PR?

File: ru-ru/python-ru.html.markdown [hunk 1]
```diff
@@ -480,17 +478,18 @@ while x < 4:
...
     x += 1  # Краткая запись для x = x + 1
 
+
 # Обрабатывайте исключения блоками try/except
 try:
     # Чтобы выбросить ошибку, используется raise
     raise IndexError("Это ошибка индекса")
 except IndexError as e:
-    pass                  # pass — это просто отсутствие оператора. Обычно здесь происходит восстановление после ошибки.
+    pass                  # pass — это просто отсутствие оператора. Обычно здесь происходит исправление ошибки.
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:487]: я-бы оставил как было, в `except` исправлять ошибки это большая редкость.

File: ru-ru/python-ru.html.markdown [hunk 2]
```diff
@@ -480,17 +478,18 @@ while x < 4:
...
 except (TypeError, NameError):
-    pass                  # Несколько исключений можно обработать вместе, если нужно.
-else:                     # Необязательное выражение. Должно следовать за последним блоком except
-    print("Всё хорошо!")  # Выполнится, только если не было никаких исключений
-finally:                  # Выполнить при любых обстоятельствах
+    pass                  # Несколько исключений можно обработать одновременно.
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:489]:
```suggestion
    pass                  # Можно обработать больше одного исключения.
```

мне кажется слово одновременно может заводить в заблуждение - обрабатывается только одно исключение.

File: ru-ru/python-ru.html.markdown [hunk 3]
```diff
@@ -434,8 +432,8 @@ for i in range(4):
     print(i)
 
 """
-"range(нижнее, верхнее)" возвращает список чисел
-от нижнего числа к верхнему
+"range(начальное, конечное)" возвращает список чисел от начального числа
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:435]:
```suggestion
"range(начало, конец)" возвращает список чисел от начального числа 
```

File: ru-ru/python-ru.html.markdown [hunk 4]
```diff
@@ -434,8 +432,8 @@ for i in range(4):
     print(i)
 
 """
-"range(нижнее, верхнее)" возвращает список чисел
-от нижнего числа к верхнему
+"range(начальное, конечное)" возвращает список чисел от начального числа 
+к конечному.
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:436]:
```suggestion
"range(начало, конец)" возвращает список чисел от начального числа 
к конечному, не включая конечное.
```

File: ru-ru/python-ru.html.markdown [hunk 5]
```diff
@@ -446,9 +444,9 @@ for i in range(4, 8):
     print(i)
 
 """
-"range(нижнее, верхнее, шаг)" возвращает список чисел
-от нижнего числа к верхнему, от нижнего числа к верхнему, увеличивая
-шаг за шагом. Если шаг не указан, значение по умолчанию - 1.
+"range(начальное, конечное, шаг)" возвращает список чисел от начального
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:447]:
```suggestion
"range(начало, конец, шаг)" возвращает список чисел от начального 
```

**Comment by ademaro** [comment]
@verhovsky, логично. Спасибо за подробное ревью.

File: ru-ru/python-ru.html.markdown [hunk 6]
```diff
@@ -509,19 +508,17 @@ with open("myfile2.txt", "w+") as file:
 # Чтение из файла
 with open('myfile1.txt', "r+") as file:
     contents = file.read()           # Читает строку из файла
-print(contents)
-# print: {"aa": 12, "bb": 21}
+print(contents)                      # => '{"aa": 12, "bb": 21}'
 
 with open('myfile2.txt', "r+") as file:
```
Comment (verhovsky) [inline ru-ru/python-ru.html.markdown:513]:
```suggestion
with open('myfile2.txt', "r") as file:
```

**Pull request merged**